### PR TITLE
feat: add `FsOptions::parse` to new `DynFs`

### PR DESCRIPTION
**Pull request by KKould**

### CODE REVIEW

File: fusio/src/lib.rs [hunk 1]
```diff
@@ -5,6 +5,7 @@ mod error;
 #[cfg(feature = "fs")]
 pub mod fs;
 pub mod local;
+pub mod parse;
```
Comment (ethe) [inline fusio/src/lib.rs:8]: should add `#[cfg(feature = "fs")]`

Comment (ethe) [reply fusio/src/lib.rs:8]: parse is not a clear mod name, parse what? place it to `src/fs/options.rs` might be a better choice.

File: fusio/src/parse.rs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        region: Option<String>,
+        sign_payload: Option<bool>,
+        checksum: Option<bool>,
+    },
+}
+
+impl FsOptions {
+    pub fn parse(self) -> Result<Arc<dyn DynFs>, Error> {
+        match self {
+            #[cfg(feature = "tokio")]
```
Comment (ethe) [inline fusio/src/parse.rs:21]: use `crate::local::LocalFs` instead.

File: fusio/src/parse.rs [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                    }
+                }
+                if let Some(region) = region {
+                    builder = builder.with_region(region);
+                }
+                if let Some(sign_payload) = sign_payload {
+                    builder = builder.with_unsigned_payload(!sign_payload);
+                }
+                if let Some(checksum) = checksum {
+                    // TODO
```
Comment (ethe) [inline fusio/src/parse.rs:55]: [with_checksum_algorithm](https://docs.rs/object_store/latest/object_store/aws/struct.AmazonS3Builder.html#method.with_checksum_algorithm)?

File: fusio/src/parse.rs (hunk 2 above)
Comment (KKould) [reply fusio/src/parse.rs]: `crate::local::LocalFs` doesn't work

Comment (ethe) [reply fusio/src/parse.rs]: why? https://github.com/tonbo-io/fusio/commit/a17d1b90613d44fc735ee762540a48c904e0b51a

**Review by ethe — APPROVED** [review]

**Pull request merged**